diagnostic.md

```diff
@@ -7,19 +7,19 @@ What command do we use to do that? Use the **correct** URL from your fork of
 this repository in your answer.
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+<!-- git clone git@github.com:acballard/git-basics-diagnostic.git. -->
 ```
 
 2.What do you do after cloning a repository, before you start making any
 changes/additions?
 
-<!-- Remove this comment and place your answer here. -->
+<!--->git commit.
 
 3.What command do we use to create a new branch? Name this branch `response`
     in your answer. Then, how do we switch to that branch?
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+<!--  git branch [branch-name]. -->
```

Comment:
3/7

Happy to go over any of this 1-on-1 We will also be covering some git and unix basics during workshop today.